aas-gui/Frontend/aas-web-gui/src/mixins/SubmodelElementHandling.ts

```diff
@@ -394,46 +394,51 @@ export default defineComponent({
             }
         },
 
-        // Get the ConceptDescriptions for the SubmodelElement from the ConceptDescription Repository
-        getConceptDescription(SelectedNode: any) {
+        // Get all ConceptDescriptions for the SubmodelElement from the ConceptDescription Repository
+        async requestConceptDescriptions(SelectedNode: any) {
```

Comment:
This seems to work but why change it? I don't see the benefit. This produces changes in all the components that use this Method. This could introduce issues if it's not tested thoroughly.

Reply:
Intention was to use a "cleaner" wording.
Previously a single ConceptDescription (singular, getConceptDescription) was retrieved, but now it is an array of several ConceptDescriptions (plural)

If you are referring to the change of "get" to "request", you're right no need for changing this!